imports/api/employee.js

```diff
@@ -0,0 +1,209 @@
+import {Meteor} from "meteor/meteor";
+import {Employees} from "./db";
+import EMPLOYEESAPI from "../constant/methods/employeesAPI";
+import { isAuthenticated } from "../util/authUtil";
+import {AuthError, EmployeeError} from "../constant/error";
+import TASKSAPI from "../constant/methods/tasksAPI";
+import {EmployeeMessage} from "../constant/message";
+
+if (Meteor.isServer) {
+    Meteor.publish("employees", () => Employees.find());
+}
+
+// username, password, accountId, employeeId
+export const insertEmployee = (db, employeeId, firstName, lastName, role) => {
```

Comment:
let's also have an email parameter when insert new employee, and change employeeId to accountId. So this line become (db, accountId, email, firstName, lastName, role)...